webapp/src/components/EtherscanLink/EtherscanLink.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+
 import { getEtherscanHref } from '@dapps/modules/transaction/utils'
+import { getContractAddress } from 'modules/wallet/utils'
 
 export default class EtherscanLink extends React.PureComponent {
   static propTypes = {
     network: PropTypes.string,
     address: PropTypes.string,
+    name: PropTypes.string,
```

Comment:
We shouldn't couple this component to this App. `EtherscanLink` is part of the `@dapps/containers`, we won't be able to use it if we do this.